src/write.rs

```diff
@@ -166,16 +173,16 @@ mod tests {
         let opt = WriteOptions::default();
         let plist_read = Value::from_file("testdata/MutatorSansLightWide.ufo/lib.plist")
             .expect("failed to read plist");
-        let tmp = TempDir::new("test")?;
+        let tmp = TempDir::new("test").unwrap();
```

Comment:
what's the motivation for these unwraps? If we're going to move to unwrapping then the test doesn't need to return a `Result`, but it seems easier to leave it as is?

Reply:
I didn't want to add a new `From` converter thing, but haven't looked at it too closely :grimacing:

Reply:
Sure, that's reasonable. Another option might be to make the return type `Box<dyn std::error::Error>`, but whatever is easiest. :)